front/pages/api/v1/w/[wId]/assistant/conversations/index.ts

```diff
@@ -83,12 +86,29 @@ async function handler(
         });
       }
 
-      const { title, visibility } = bodyValidation.right;
+      const { title, visibility, message } = bodyValidation.right;
 
       const conversation = await createConversation(auth, {
         title,
         visibility,
       });
+
+      if (message) {
+        // Not awaiting this promise on prupose. We want to answer "OK" to the client ASAP and process
+        // the events in the background.
+        void postUserMessageWithPubSub(auth, {
```

Comment:
we have changed this to await for user_new_message in postMessage
We should do that here too probably?

Reply:
Not here as this is a conversation creation endpoint. This is just a way to bundle the new message in the same call

Reply:
This non-waiting is aligned with the conversation creation with message imlpementation in our internal API